src/lsdb/catalog/catalog.py

```diff
@@ -47,6 +48,14 @@ def __init__(
         super().__init__(ddf, hc_structure)
         self._ddf_pixel_map = ddf_pixel_map
 
+    def get_pixels(self) -> pd.DataFrame:
```

Comment:
So in hipscat we have `get_pixels` and `get_healpix_pixels` which is a bit of a mess. I'd like both those methods available here, but I don't like that naming. I think it makes more sense to call this `get_partition_info` so we can use `get_pixels` to get the healpix pixels.

Reply:
If you wanted to also add `get_pixels` calling `get_healpix_pixels` in this pr, that would be amazing, but that can be left for another pr

Reply:
But then again `get_partition_info` sounds like it will get the partition info object. I kinda just hate the whole df.

Reply:
You know what, leave it, this isn't an issue for this pr, I'll make another issue for it.